packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -101,7 +107,7 @@ export class SupportComponent {
 	 * @returns {string} Title of the article.
 	 */
 	async getOpenArticleTitle(): Promise< string > {
-		const locator = this.anchor.getByRole( 'article' ).getByRole( 'heading' ).nth( 0 );
+		const locator = this.anchor.getByRole( 'article' ).getByRole( 'heading' ).first();
```

Comment:
This effectively changes nothing, because the TypeScript transpiler already converts the `.nth(0)` command to a `.first()`.